metric/syncfloat64.go

```diff
@@ -78,6 +85,13 @@ type Float64UpDownCounter interface {
 	// Use the WithAttributeSet (or, if performance is not a concern,
 	// the WithAttributes) option to include measurement attributes.
 	Add(ctx context.Context, incr float64, options ...AddOption)
+
+	// IsEnabled returns true if this instrument is enabled.
+	//
+	// This helps users avoid performing computationally expensive operations when
+	// recording measurements. The returned value cannot be cached since it
```

Comment:
```suggestion
	// recording measurements. The returned value should not be cached since it
```

felt this is more apt, but feel free to ignore this comment, as I don't have full context on Otel Go!